src/service/vsync-impl.js

```diff
@@ -150,12 +151,21 @@ export class Vsync {
 
     /** @private {!JankMeter} */
     this.jankMeter_ = new JankMeter(this.win);
-  }
 
-  /** @private */
-  onVisibilityChanged_() {
-    if (this.scheduled_) {
-      this.forceSchedule_();
+    /** @private @const {!MutationObserver|null} */
```

Comment:
`?MutationObserver`

Also please add a brief comment explaining what this does and why.